src/taco-remote-lib/ios/ios.ts

```diff
@@ -218,6 +218,15 @@ class IOSAgent implements ITargetPlatform {
                 res.status(200).json(buildInfo.localize(req, resources));
             }
         });
+
+        ideviceinstaller.on("error", function (err: any): void {
+            res.status(500);
+            if (err.code === "ENOENT") {
+                res.send(resources.getStringForLanguage(req, "IdeviceinstallerNotFound"));
```

Comment:
The name of the tool is "ideviceinstaller", so I went with "IdeviceinstallerNotFound", but it's easy enough to change. This isn't something that you broke earlier on, don't worry about that :)